modules/Hlsjs/resources/hlsjs.js

```diff
@@ -473,10 +473,10 @@
 			/**
 			 * Override player callback after changing media
 			 */
-			changeMediaCallback: function(){
-				this.getPlayer().play();
```

Comment:
@OrenMe - thinking of it, we must verify playlist auto continue when using hlsjs after removing the changeMediaCallback call. Might get broken...